src/plugins/wizard/public/application/components/workspace.tsx

```diff
@@ -67,9 +70,20 @@ export const Workspace: FC = ({ children }) => {
         ) : (
           <EuiFlexItem className="wizWorkspace__empty">
             <EuiEmptyPrompt
-              iconType="visBarVertical"
-              title={<h2>Welcome to the wizard!</h2>}
-              body={<p>Drag some fields onto the panel to visualize some data.</p>}
+              title={<h2>Drop some fields here to start</h2>}
+              body={
+                <Fragment>
+                  <p>Drag a field directly to the canvas or axis to generate a visualization.</p>
+                  <span className="wizWorkspace__container">
+                    <p className="wizWorkspace__fieldSvg">
+                      <EuiIcon type={fields_bg} size="original" />
```

Comment:
you can directly add a `className` prop to `EuiIcon`. You don't need to wrap the icon with additional DOM elements for just that :)